deploy/kubernetes/kiali-secrets.yaml

```diff
@@ -8,4 +8,4 @@ metadata:
 type: Opaque
 data:
   username: YWRtaW4=    # admin
-  password: YWRtaW4=    # admin
+  passphraseKey: YWRtaW4=    # admin
```

Comment:
It should be "passphrase" is we are trying to match that of upstream Istio: https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/kiali/values.yaml#L24

You confused the helm variable with the actual name used.

Reply:
Sorry! Fixed in 4509352.